pkg/run/run.go

```diff
@@ -122,12 +134,12 @@ func (r *Runtime) Run() error {
 			return r.Stop(nil)
 
 		case <-timeoutTimer.C:
-			glog.Warningf("Timeout %s reached, stopping ...", r.runTimeout.String())
-			return r.Stop(fmt.Errorf("timeout reached during run phase: %s", r.runTimeout.String()))
+			glog.Warningf("Timeout %s reached, stopping ...", r.conf.RunTimeout.String())
+			return r.Stop(fmt.Errorf("timeout reached during run phase: %s", r.conf.RunTimeout.String()))
 
 		case <-probeTick.C:
 			_, err := r.probeUnitStatuses()
-			if err != nil {
+			if err != nil && !r.conf.SkipProbe {
```

Comment:
What are the challenges with just having this as the first check when the probe ticker is fired.

```go
if r.conf.SkipProbe {
   continue
}
```

Reply:
I thought about that in the first place but I thought useful to still have the errors prompted to the terminal.

Reply:
Isn't `skip-probes` opting to ignore errors entirely from probing?

Reply:
Indeed, I'll change that and we'll see if it's missing in the future.